src/components/rhelGraphCard/__tests__/__snapshots__/rhelGraphCard.test.js.snap

```diff
@@ -246,125 +246,125 @@ exports[`RhelGraphCard Component should render a non-connected component: non-co
 exports[`RhelGraphCard Component should render multiple states: error shows zeroed bar values 1`] = `
 Object {
   "chartBarData": Array [
+    Object {
```

Comment:
i'm good after this bit is updated... i believe it should start w/ Jun 1 if Jun 1 UTC is passed.

Reply:
ugh missed that

Reply:
ahh... yes, you are correct:

```
new Date('2019-06-01T00:00:00Z')
Fri May 31 2019 20:00:00 GMT-0400 (Eastern Daylight Time)
```

Ok, yes, if sourced from the server....shouldn't matter. I am guessing that Jest uses the system UTC time (so EST in my case), but not positive.

Maybe some way to mock this better in the future? FWIW, it's not obvious. Could also just set to something same day in UTC /U.S. time zones for being more obvious...

```
new Date('2019-06-01T12:00:00Z')
Sat Jun 01 2019 08:00:00 GMT-0400 (Eastern Daylight Time)
```